daemon/api_sideload_n_try.go

```diff
@@ -219,7 +219,7 @@ func sideloadManySnaps(st *state.State, snapFiles []*uploadedSnap, flags sideloa
 		userID = user.ID
 	}
 
-	tss, err := snapstateInstallPathMany(context.TODO(), st, sideInfos, tempPaths, userID, &flags.Flags)
+	tss, err := snapstateInstallPathMany(context.TODO(), st, sideInfos, tempPaths, userID, &flags.Flags, &snapstate.ManyFlags{})
```

Comment:
I think it would be ok to allow `nil` flags in these functions (then, inside the function set it to `&snapstate.ManuFlags{}` if nil was passed to avoid repeated nil checks in the logic).